packages/dx-scheduler-core/src/types/scheduler-core.types.ts

```diff
@@ -76,3 +76,11 @@ export type FormatDateTimeGetterFn = (locale: string | string[]) => FormatterFn;
 export type DateTimeFormatInstanceFn = (
   locale: string | string[], formatOptions: Intl.DateTimeFormatOptions,
 ) => Intl.DateTimeFormat;
+
+/** Describes a current view object. */
+export interface CurrentView {
```

Comment:
I think we should consider other type names too, because this type is used for the `availableViews: CurrentView[]` getter and it goes against a semantic meaning: it's assumed that there's only a single 'current' or 'active' view.

Reply:
What about View? Or ViewProperties? Or ViewCharacteristics?

Reply:
May be `SchedulerView`

Reply:
I agree with this name